client-app/shared/layout/composables/useSearchBar.ts

```diff
@@ -58,10 +58,7 @@ export default (
   }
 
   async function showSearchBar(): Promise<void> {
-    if (searchBarVisible.value) {
-      return;
-    }
-    searchBarVisible.value = true;
+    searchBarVisible.value = !searchBarVisible.value;
```

Comment:
either rollback, or here `= true`

Reply:
Expected result: search button is still visible, click should toggle the searchbar.

Reply:
Will refactor after Nuxt